Sources_v3/Database/DTOs/ChannelDTO_Tests.swift

```diff
@@ -344,6 +344,34 @@ class ChannelDTO_Tests: XCTestCase {
         )
     }
     
+    func test_channelUreadCount_calculatedCorrectly() {
+        // Create and save a current user, to be used for channel unread calculations
+        let currentUserPayload = dummyCurrentUser
+        
+        database.write {
+            try! $0.saveCurrentUser(payload: currentUserPayload)
+        }
+        
+        let channelId: ChannelId = .unique
+        
+        let payload = dummyPayload(with: channelId)
+        
+        // Asynchronously save the payload to the db
+        database.write { session in
+            try! session.saveChannel(payload: payload)
+        }
```

Comment:
we have test helpers like `database.createCurrentUser()` and `database.createChannel(cid: <>)`, maybe the can help? I'm not sure because of the specific dummy payloads you use here.

Reply:
I've refactored it to use `createCurrentUser(id:)` in 6ffa3d0ad407a90560218ef9a281f902d4b042fe, but I can't use `createChannel(cid:)` since I need `dummyChannelRead` and `dummyMessage` to be in the channel